ssl/tls13_enc.c

```diff
@@ -764,12 +772,132 @@ int tls13_change_cipher_state(SSL *s, int which)
         s->statem.enc_write_state = ENC_WRITE_STATE_WRITE_PLAIN_ALERTS;
     else
         s->statem.enc_write_state = ENC_WRITE_STATE_VALID;
+#ifndef OPENSSL_NO_KTLS
+# if defined(OPENSSL_KTLS_TLS13)
+#  ifndef __FreeBSD__
+    if (!(which & SSL3_CC_WRITE)
```

Comment:
So we only support transmit for TLSv1.3?

Reply:
Yes, right now I'm not sure how to correctly implement RX path for TLSv1.3 because it have a lot of dependencies used in TLSv1.2 code. I'll try it later in another PR.